### PR TITLE
Added SQL commands to Database object

### PR DESCRIPTION
**Pull request by majulahsingapuri**

### CODE REVIEW

File: capstone/src/main/java/capstone/Objects/Customer.java [hunk 1]
```diff
@@ -1,38 +1,179 @@
...
+  private String phoneNumber;
 
   /**
-   * Initialiser for Class. Creates new object with new username, default password and Customer
-   * AccessLevel.
+   * Initialiser for Customer Class,
    *
+   * @param id the User's id
    * @param username The User's username.
+   * @param password The User's password.
```
Comment (SuperiormonLBJ) [inline capstone/src/main/java/capstone/Objects/Customer.java:40]: comment error?  should not be all password for other param

File: capstone/src/main/java/capstone/Objects/Customer.java
Comment (cori-hwf) [inline capstone/src/main/java/capstone/Objects/Customer.java:1]: implement getCustomerTransaction method

File: capstone/src/main/java/capstone/Objects/Database.java
Comment (cori-hwf) [inline capstone/src/main/java/capstone/Objects/Database.java:1]: implement getCustomerTransaction api

File: capstone/src/main/java/capstone/Views/LoginView.java
Comment (cori-hwf) [inline capstone/src/main/java/capstone/Views/LoginView.java:1]: To check the optional object result before .get()

**Review by SuperiormonLBJ — APPROVED** [review]

**Review by RoyTeoh — APPROVED** [review]

File: capstone/src/main/java/capstone/Views/LoginView.java
Comment (majulahsingapuri) [reply capstone/src/main/java/capstone/Views/LoginView.java:1]: this is to be implemented on the View side. this hotfix was to prevent any error messages

**Review by cori-hwf — APPROVED** [review]

**Pull request merged**